pkgs/tools/filesystems/zerofree/default.nix

```
@@ -9,12 +14,14 @@ stdenv.mkDerivation rec {
     sha256 = "0rrqfa5z103ws89vi8kfvbks1cfs74ix6n1wb6vs582vnmhwhswm";
   };
 
-  buildInputs = [ e2fsprogs ];
+  buildInputs = [ e2fsprogs installShellFiles ];
 
   installPhase = ''
-    mkdir -p $out/bin
+    mkdir -p $out/bin $out/share/zerofree
     cp zerofree $out/bin
-'';
+    cp COPYING $out/share/zerofree
```

```suggestion
    cp COPYING $out/share/zerofree/COPYING
```

Did you mean this? Also I don't feel comfortable to fetch a manpage from Ubuntu.